lib/JSON/Validator.pm

```diff
@@ -1307,7 +1307,7 @@ on how a C<$ref> looks before and after:
   {"$ref":"http://example.com#/foo/bar"}
      => {"$ref":"#/definitions/_http___example_com-_foo_bar"}
 
-=item * C<{schema => {...}}>
+=item * C< {schema => {...}} >>
```

Comment:
There seems to be a typo here? (Missing <)

Also, is the white space requires about {}?

Reply:
Typo fixed! sorry about that.

Yes, the tags are `C<< ` and ` >>C`. The whitespace isn't visible. https://perldoc.perl.org/perlpod.html#Formatting-Codes